packages/pages/src/dev/dev.ts

```diff
@@ -25,6 +26,12 @@ const handler = async ({
   noGenFeatures,
   port,
 }: DevArgs) => {
+  if (!fs.existsSync("config.yaml")) {
+    console.warn(
+      "config.yaml not found. Please run 'npx pages upgrade' to create it."
```

Comment:
Can you check what the PMs want the warning message to be? I could see them wanting something more like "It looks like you're using an older setup. Please run 'npx pages upgrade' to upgrade to the latest format."